src/JBrowse/Model/BAIIndex.js

```diff
@@ -112,6 +112,38 @@ return declare( TabixIndex, {
         deferred.resolve();
     },
 
+    featureCount: function(tid) {
+        var index = this.indices[tid];
+        if (!index) {
+            return -1;
+        }
+        var p = 4;
+        var nbin = readInt(index, 0);
+        var overlappingBins = function() {
+            var intBins = {};
+            var intBinsL = [this._bin_limit()+1];
+            for (var i = 0; i < intBinsL.length; ++i) {
+                intBins[intBinsL[i]] = true;
+            }
+            return intBins;
+        }.call(this);
```

Comment:
no, use a regular block context or something.  no more function() {}.call(this) to get block scoping.